### PR TITLE
FEATURE/MINOR: Adds http-request return

### PR DESCRIPTION
**Pull request by schegi**
* adds actions.Return
* adds tests

### CODE REVIEW

File: parsers/http/actions/return.go [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+	Name string
+	Fmt  string
+}
+
+func IsPayload(in string) bool {
+	_, ok := map[string]struct{}{
+		"string":    {},
+		"lf-string": {},
+		"file":      {},
+		"lf-file":   {},
```
Comment (oktalz) [inline parsers/http/actions/return.go:49]: creating this map every time that `http-request return` gets parsed will
put a lot of pressure on GC if many `http-request return ...` lines exists.

I think its better to have this a global (unexported/private) const

File: parsers/http/actions/return.go [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+		404: {},
+		405: {},
+		408: {},
+		410: {},
+		413: {},
+		425: {},
+		429: {},
+		500: {},
+		503: {},
+		504: {},
```
Comment (oktalz) [inline parsers/http/actions/return.go:68]: same comment, global (unexported/private) const would be better here

**Review by oktalz — CHANGES_REQUESTED** [review]

**Comment by oktalz** [comment]
hi @schegi 
PR seems good, I'm just worried a bit what will happen if large amount of lines needs to be parsed.
beside comments I made it looks really good

File: parsers/http/actions/return.go (hunk 2 above)
Comment (schegi) [reply parsers/http/actions/return.go]: Done

File: parsers/http/actions/return.go (hunk 1 above)
Comment (schegi) [reply parsers/http/actions/return.go]: done

**Pull request merged**